src/gc-arena/src/types.rs

```diff
@@ -6,8 +6,21 @@ use crate::collect::Collect;
 
 #[derive(Copy, Clone, Eq, PartialEq, Debug)]
 pub(crate) enum GcColor {
+    /// An object that was not reached during tracing.
```

Comment:
I know I'm being super pedantic but technically it is an object that might have not been reached by tracing *yet*, if we are in phases that do tracing.

Reply:
I've updated the comment